prov/rxm/src/rxm_ep.c

```diff
@@ -339,26 +339,37 @@ static int rxm_ep_txrx_res_open(struct rxm_ep *rxm_ep)
 	dlist_init(&rxm_ep->post_rx_list);
 	dlist_init(&rxm_ep->repost_ready_list);
 
+	ret = rxm_buf_pool_create(rxm_ep,
+				  rxm_ep->msg_info->tx_attr->size,
+				  rxm_ep->rxm_info->tx_attr->inject_size +
+				  sizeof(struct rxm_rma_buf),
+				  &rxm_ep->buf_pools[RXM_BUF_POOL_RMA],
+				  RXM_BUF_POOL_RMA);
+	if (ret)
+		goto err5;
```

Comment:
This function needs to go on a diet.  Please consider moving all the pool allocations into a new function.

Reply:
:) Yep, should be refactored